tests/jerry/func-decl.js

```diff
@@ -23,3 +23,27 @@ assert ((function() {
     }
     return f();
 })() === 'bar');
+
+function check_syntax_error (s) {
```

Comment:
No, it is incorrect merge. The function is from another patch.
That patch was already merged, so I'll rebase this pull request.